local-notifications/README.md

```diff
@@ -20,7 +20,9 @@ Starting on Android 12, scheduled notifications won't be exact unless this permi
 <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM" />
 ```
 
-Note that even if the permission is present, users can still disable exact notifications from the app settings.
+Note that even if the permission is present, users can still disable exact notifications from the app settings.  Use `checkExactNotificationSetting()` to check the the value of the setting.  If a user disables this setting, the app will restart and any notification scheduled with an exact alarm will be deleted.  If your application depends on exact alarms, be sure to check this setting in order provide fallbacks or alternative behavior.
```

Comment:
the "be sure to check this setting" should mention something like in "app resume", or in `App.appStateChange` event, or on app launch since the app is being restarted if the setting changes.